app/assets/javascripts/admin/addon/components/admin-site-settings-filter-controls.gjs

```diff
@@ -42,6 +42,9 @@ export default class AdminSiteSettingsFilterControls extends Component {
 
   @action
   runInitialFilter() {
+    document
+      .querySelector(".admin-site-settings-filter-controls__input")
+      .focus();
```

Comment:
I think using our auto-focus modifier on the element would be preferable to querying for it.

Reply:
I thought this was shit, I didn't realise we had a modifier :eyes: